src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -9083,11 +9084,11 @@ TEST_P(convolution_gpu_onednn, conv_onednn_cases) {
     for (auto& p : network.get_primitives_info())
         std::cerr << p.original_id << " " << p.kernel_id << std::endl;
 
-    auto out_mem = network.get_output("conv_fsv").get_memory();
-    mem_lock<FLOAT16> out_ptr{ out_mem, get_test_stream() };
-    auto out_lay = out_mem->get_layout();
+    auto out_ptr = network.get_output_values<FLOAT16>("conv_fsv");
+    auto out_lay = network.get_output_layout("conv_fsv");
+    ASSERT_TRUE(is_eq_data_type_T<FLOAT16>(out_lay.data_type));
```

Comment:
why do we need this function? What about just checking `ASSERT_EQ(out_lay.data_type, data_types::f16)`?

Reply:
Reverted related changes.